ledger/complete/ledger.go

```diff
@@ -342,6 +342,10 @@ func (l *Ledger) ExportCheckpointAt(
 		return ledger.State(hash.DummyHash), fmt.Errorf("constructing updated trie failed: %w", err)
 	}
 
+	statecommitment := ledger.State(newTrie.RootHash())
+
+	l.logger.Info().Msgf("successfully built new trie. New statecommitment: %v", statecommitment.String())
```

Comment:
log the statecommitment as soon as it's generated, so that we can continue generating root block earlier